RELEASE_PROCESS.rst

```diff
@@ -0,0 +1,39 @@
+Prepare for the Release
+-----------------------
+
+Clone a fresh copy from the root repo. Do not attempt a release in your
+current working repository as the following commands expect a fresh clone.
+
+::
+
+     git clone --branch master git@github.com:juju/juju-gui.git
```

Comment:
This is confusing...where are these lines from? Maybe delete these lines and update the phrasing to be:

```
Clone a fresh copy from the root repo. Do not attempt a release in your
current working repository as the charm release commands expect a fresh clone.
```